openmc/deplete/results_list.py

```diff
@@ -58,8 +58,8 @@ def get_atoms(self, mat, nuc, nuc_units="atoms", time_units="s"):
 
         Parameters
         ----------
-        mat : str
-            Material name to evaluate
+        mat : openmc.Material
```

Comment:
I think for now we should change it to accept either the ID string as before _or_ a material instance.

Reply:
thanks Paul, good idea this makes it easier to pass the tests and means this is a non breaking change